src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -100,8 +115,78 @@ class AmendmentTableImpl : public AmendmentTable
 void
 AmendmentTableImpl::addInitial ()
 {
-    // For each amendment this version supports, construct the AmendmentState object by calling
-    // addKnown. Set any vetoes or defaults. A pointer to the AmendmentState can be stashed
+    std::set<std::pair<uint256, std::string>> toAdd;
+
+    for (auto const& _ : detail::PreEnabledAmendments)
```

Comment:
I don't see the advantage of breaking with almost fifty years of tradition in C and C++ of just using a single letter `i`, `a` or the like for loop iterators.

And even though all the compilers I know accept it, it's technically wrong - namespaces starting with an `_` are reserved for the global namespace, which this is not! 

http://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier